pkgs/development/tools/misc/semver-tool/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "semver-tool-${version}";
+  version = "2.1.0";
+
+  src = fetchFromGitHub {
+    owner = "fsaintjacques";
+    repo = "semver-tool";
+    rev = version;
+    sha256 = "0lpwsa86qb5w6vbnsn892nb3qpxxx9ifxch8pw3ahzx2dqhdgnrr";
+  };
+
+  buildPhase = "true"; # otherwise we try to 'make' which fails.
```

Comment:
This is supposed to have the effect to disable the `buildPhase`?
If so then you should be able to use `dontBuild = true;`.

Reply:
`dontBuild = true;`. Typical nixers. Why not `build = false;`?

Reply:
> Why not `build = false;` ?

And `configure = false` would be better too.

We also use `dontConfigure` but it actually doesn't exist yet and were just hoping that it will someday.